Makefile

```diff
@@ -32,10 +32,12 @@ deps-update:
 	go mod tidy && \
 	go mod vendor
 
-functests: label-worker-rt feature-deploy
+functests:
 	@echo "Running Functional Tests"
 	FEATURES="$(FEATURES)" hack/run-functests.sh
 
+functests-on-ci: setup-test-cluster feature-deploy feature-wait functests
```

Comment:
also, we need to make a pr to change the prow config to use this make target. that probably needs to be committed with this change.

Reply:
Yes I was aware of that. But still, I preferred to change the names to be more readable.